client/src/components/Login.tsx

```diff
@@ -177,19 +180,32 @@ const LoginWithoutI18n = (props: LoginProps) => {
             </Trans>
           </span>
         )}
+        {isLoginStep && (
+          <LocaleLink
```

Comment:
is there a reason why you used `LocaleLink` instead of `Link` from the component library here?

Reply:
it does use it within LocaleLink with the `component={JFCLLink}` so it gets the styles, but I used LocaleLink so that it uses the proper `/en` `/es` on that link, which I believe we do everywhere in wow

Reply:
i guess it would actually be better to just edit the LocaleLink to always use the jfcl component like this to simplify all our uses of it since this is kinda gross currently

Reply:
oh and does this cover [sc-13974](
https://app.shortcut.com/justfixnyc/story/13974/move-forgot-password-link-out-of-the-input-field?vc_group_by=day&ct_workflow=all&cf_workflow=500000153)?

Reply:
yeah I think editing LocaleLink would be cleaner! we can make a ticket for it though unless you wanna include it in this pr

Reply:
Hmm actually I don't know if we're ready to use that in every location yet and it might take some time to review everything, I'll make a ticket for this to do later though

Reply:
oh yes, missed that ticket, thanks